setup.cfg

```diff
@@ -38,12 +38,25 @@ package_dir = =src
 packages = find:
 
 [options.extras_require]
+# Used to run tests with minimal dependencies.
 test =
     cython-test-exception-raiser ~= 1.0
     PyHamcrest >= 1.9.0
+# Use to build the narrative, api and read the docs documentation.
+docs =
+    pydoctor==20.7.2
+    sphinx~=3.2
+    sphinx-rtd-theme~=0.5.0
+    pillow~=7.2
+    readthedocs-sphinx-ext~=2.1
+    recommonmark~=0.6.0
+
 dev =
+    %(test)s
+    %(docs)s
+    %(tls)s
```

Comment:
I think the `tls =` lines should be above the `%(tls)s` line

Reply:
I tried  all this and they work... I don't think that the order . I put them in dev as I think that developers wants them 99% of the time.

```
pip install -e .[tls]
pip install -e .[dev]
pip install -e .[docs]
```

Reply:
oh I just think it's less confusing to have the ConfigParser keys in topological order